modules/storage-rocksdb/src/main/java/org/apache/ignite/internal/storage/rocksdb/index/RocksDbHashIndexStorage.java

```diff
@@ -145,20 +161,46 @@ public void remove(IndexRow row) {
         }
     }
 
+    @Override
+    public void destroy() {
+        byte[] rangeEnd = rangeEnd(constantPrefix);
+
+        try (WriteOptions writeOptions = new WriteOptions().setDisableWAL(true)) {
+            // If we can't construct the range end prefix (nearly impossible case), we need to fallback to manually removing the data,
+            // since 'deleteRange' doesn't work without the upper bound.
+            if (rangeEnd == null) {
```

Comment:
I'd rather have an assertion that this is impossible. Partition id stops way before 0xffff.
By the way, have you updated the list of column families in `RocksDbTableStorage#scheduleFlush`? It's easy to miss, because I didn't document it :(

Reply:
> By the way, have you updated the list of column families in RocksDbTableStorage#scheduleFlush

Wow, this is a very nice hint